packages/theme/src/theme/_forms.radio.scss

```diff
@@ -59,7 +59,7 @@ $tt-radio-hover-border-color: tint($lochmara, 15) !default;
 
 		&:after,
 		+ *:after {
-			@include checkbox-outline;
+			@include outline;
```

Comment:
Should include border-radius too? 
Guidelines are not enough sufficient on that point 🤔